README.md

```
@@ -68,89 +69,92 @@
-  pnpm set version 버전 # 프로젝트 최상위 폴더 위치에서 명령어 입력
-  ```
-
-  - **pnpm 명령어 예시**
-  ```
-  pnpm install # 전체 설치
-  pnpm add 라이브러리 # 라이브러리 설치
-  pnpm dev # 실행
-  ```
+- **pnpm 버전 설치 방법**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)